substratevm/src/com.oracle.svm.core.graal.llvm/src/com/oracle/svm/core/graal/llvm/util/LLVMIRBuilder.java

```diff
@@ -78,6 +98,11 @@ public LLVMIRBuilder(LLVMIRBuilder primary) {
         this.context = primary.context;
         this.builder = LLVM.LLVMCreateBuilderInContext(context);
         this.module = primary.module;
+        if (LLVMOptions.IncludeLLVMSourceDebugInfo.getValue()) {
```

Comment:
This constructor is only used to create tiny helper methods that represent a single instruction in a main program method (e.g. a compare-and-swap between two objects). Does it make sense to emit debug info here as well? There wouldn't be anything more specific than the debug info attached to the call to this helper method in the main (primary) builder.

Reply:
I see, I removed debug info emission for this constructor